src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -355,33 +355,32 @@ protected AnalyzerDriver(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerM
             }
         }
 
-        private async Task ProcessEventAsync(CompilationEvent e, CancellationToken cancellationToken)
+        private static readonly Task s_emptyTask = Task.FromResult<object>(null);
```

Comment:
I couldn't find the equivalent of the Workspace layer's "SpecializedTasks.Empty" in the compiler layer. Is there one?